java/libphonenumber/src/com/google/i18n/phonenumbers/ShortNumberInfo.java

```diff
@@ -544,12 +543,12 @@ private boolean matchesEmergencyNumberHelper(String number, String regionCode,
       return false;
     }
     PhoneMetadata metadata = MetadataManager.getShortNumberMetadataForRegion(regionCode);
-    if (metadata == null || !metadata.hasEmergency()) {
+    if (metadata == null || metadata.emergency == null) {
```

Comment:
Are you sure the default PhoneNumberDesc is null? Or is it a PhoneNumberDesc with all the fields set to their defaults?